api/utils/sshutils/conn.go

```diff
@@ -105,3 +119,23 @@ func (d *DialReq) CheckAndSetDefaults() error {
 	}
 	return nil
 }
+
+func (d *DialReq) ClientSrcNetAddr() net.Addr {
+	return getNetAddr(d.ClientSrcAddr)
+}
+
+func (d *DialReq) ClientDstNetAddr() net.Addr {
+	return getNetAddr(d.ClientDstAddr)
+}
```

Comment:
`net.Conn` names for addresses are more direction agnostic, since it depends from which side you're looking at it. Here I try to be more precise so direction is clear, it also is consistent with naming in the PROXY protocol, where it ultimately comes from.

Reply:
We already have `ClientSrcAddr`/`ClientDstAddr` of type `string` on the struct, these functions are for convenience of the callers to get `net.Addr` so they don't have to parse it themselves. I've added documentation 👍 
Yes, I think at this moment we should return nil if we can't get correct IP and caller is responsible for handling it.